js/base/Exchange.js

```diff
@@ -2085,4 +2086,18 @@ module.exports = class Exchange {
         params = this.omit (params, 'type');
         return [ type, params ];
     }
+
+    yearMonthDay (dateStr) {
```

Comment:
We don't need this, cause we have yyyymmdd, yymmdd, ymd in the base classes.

Reply:
`yyyymmdd` returns `2021-12-31` instead of `20211231`

Reply:
@samgermain it has the secont `infix` argument for that, it's a dash `-` by default, but you can set it to `-`:

- https://github.com/ccxt/ccxt/blob/master/js/base/functions/time.js#L161